File: find-the-highest-altitude/find-the-highest-altitude.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        n = len(gain)
        ans = []
        ans.append(0)

        for i in range(0, n):
            check = (ans[i] + gain[i])

            ans.append(check)

        mx = max(ans)

        return mx

File: 1833-find-the-highest-altitude/find-the-highest-altitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int pre = 0, ans = 0;

        foreach (int g in gain) {
            pre += g;

            if (pre > ans) {
                ans = pre;
            }
        }

        return ans;
    }
}